server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -389,6 +389,12 @@
     <to last="true">/spark/admin/plugins</to>
   </rule>
 
+  <rule>
+    <name>Elastic Agent Configurations SPA</name>
+    <from>^/admin/elastic_profiles(/?)$</from>
+    <to last="true">/spark/admin/elastic_agent_configurations</to>
```

Comment:
This looks like an internal rewrite, not a redirect.

Reply:
See http://tuckey.org/urlrewrite/manual/4.0/#to

`<to last="true" type="temporary-redirect" />`

Reply:
Sure will update this

Reply:
@ketan -- just  a question regarding `permanent-redirect` (301) vs `temporary-redirect` (302).

As the old URL is moved permanently and will be removed (in near future), should we use `permanent-redirect`?